db/schema.rb

```diff
@@ -70,12 +70,6 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
-  create_table "bulk_edits", force: :cascade do |t|
```

Comment:
I'm surprised you're removing `bulk_edits` from the schema. Is that peaceful?

Reply:
Yeah I'm not sure where this came from... maybe from one of my rebases. I will remove it

Reply:
@alaroia-cfa do I have to revert the rebase or can I just copy-pasta it back in?

Reply:
Copy-pasta is fine. If you are going to be doing a squash merge in GitHub, then the only important thing is the diff.